Dockerfile

```diff
@@ -21,6 +21,8 @@ ADD docker/docs.conf /etc/nginx/conf.d/docs.conf
 
 COPY --from=builder /tmp/docs.tar.gz /tmp/docs.tar.gz
 
+EXPOSE 80
```

Comment:
Interesting. I expected this to be inherited. ref https://hub.docker.com/layers/nginx/library/nginx/1.13.9/images/sha256-ed3c34093c4503d49abf8f5b67daafce83705277d9a69cde5b2a3dd8c71c899e?context=explore

Reply:
Ahh. I didn't think to check the parent. I _think_ it's still inherited?